scripts/Configure-Environment.ps1

```diff
@@ -74,15 +70,3 @@ Write-Host "Using the following GalleryBaseUrl: $GalleryBaseUrl"
 $env:GalleryBaseUrl = $GalleryBaseUrl
```

Comment:
Yes, that's what I wrote in the PR comments. I will do it as part of this work after the PR is completed